### PR TITLE
More informative errors

### PR DESCRIPTION
**Pull request by angrymouse**

### CODE REVIEW

File: src/core/modules/impl/DefaultCreateContract.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export class DefaultCreateContract implements CreateContract {
         transfer
       });
     } else {
-      throw new Error('Unable to write Contract Source.');
+      throw new Error('Unable to write Contract Source.\n'+resource.statusText.msg);
```
Comment (ppedziwiatr) [inline src/core/modules/impl/DefaultCreateContract.ts:74]: hmm. `resource`? did you mean `response`?
Also - the `statusText` is defined as a string (so there is no "msg" property):
```
post(transaction: Transaction | Buffer | string | object): Promise<{
        status: number;
        statusText: string;
        data: any;
    }>;
```

How about changing to 
```
`Unable to write Contract Source.\n${response?.statusText}`
```

File: src/core/modules/impl/DefaultCreateContract.ts [hunk 2]
```diff
@@ -115,7 +115,7 @@ export class DefaultCreateContract implements CreateContract {
     if (response.status === 200 || response.status === 208) {
       return contractTX.id;
     } else {
-      throw new Error('Unable to write Contract Initial State');
+      throw new Error('Unable to write Contract Initial State\n'+resource.statusText.msg);
```
Comment (ppedziwiatr) [inline src/core/modules/impl/DefaultCreateContract.ts:118]: same here...

**Review by ppedziwiatr — CHANGES_REQUESTED** [review]

File: src/core/modules/impl/DefaultCreateContract.ts (hunk 2 above)
Comment (angrymouse) [reply src/core/modules/impl/DefaultCreateContract.ts:118]: Yes, exactly

**Review by angrymouse — COMMENTED** [review]
resource means response

**Pull request merged**